investor_wallet_platform_base/views/menus.xml

```diff
@@ -14,6 +14,7 @@
     </record>
 
     <menuitem name="Structures" id="menu_wallet_platform_structure" parent="easy_my_coop.menu_easy_my_coop_config" action="action_wallet_platform_structure_form" groups="group_platform_wallet_manager" sequence="401"/>
+    <menuitem name="Structures to validate" id="menu_wallet_platform_structure_to_validate" parent="easy_my_coop.menu_easy_my_coop_config" action="action_iwp_structure_to_validate_form" groups="group_platform_wallet_manager" sequence="402"/>
```

Comment:
group should be `easy_my_coop.group_easy_my_coop_manager`

Reply:
nope as this is a menu for iwp manager to see which structures are waiting for a validation